browser.go

```
@@ -111,27 +147,50 @@ func login(fb *feedBrowser) stateFunc {
 }
 
 func navigateToLikedTweets(fb *feedBrowser) stateFunc {
-	fb.page.
-		MustNavigate(fmt.Sprintf("%s/%s/likes", fb.baseUrl, fb.username)).
-		MustWaitLoad().
-		MustWaitIdle()
+	hjRouter := fb.page.HijackRequests()
+	hjRouter.Add(
```

golangci-lint (1.62.2)

Error return value of `hjRouter.Add` is not checked

(errcheck)